experimental/addons/postgres-operator/definitions/traits/postgres-expose.cue

```diff
@@ -0,0 +1,80 @@
+"postgres-expose": {
+	alias: ""
+	annotations: {}
+	attributes: {
+		workload: type: "autodetects.core.oam.dev"
+	}
+	description: "postgres expose trait"
+	labels: {}
+	type: "trait"
+}
+
+template: {
+	outputs: service: {
+		apiVersion: "v1"
+		kind:       "Service"
+		metadata: annotations: parameter.annotations
+		spec: {
+			selector: {
+				"application":  "spilo"
+				"cluster-name": "postgres"
+				"spilo-role":   "master"
+				"team":         "acid"
+			}
```

Comment:
I had the same concern regarding with @wonderflow . It seems that the existing Expose trait can accomplish the same functionality. Have you considered using it instead

Reply:
@wangyikewxgm @wonderflow I tried to do so, But I'm getting an error
```
could not create cluster: could not create master endpoint: endpoints "postgres" already exists
```
it seems like expose trait is getting created first then `postgres cluster` using built in expose trait.